workflows/sarek.nf

```diff
@@ -1001,6 +1008,64 @@ workflow SAREK {
             rt_file
         )
 
+        if (params.concatenate_vcfs) {
+            // Concatenate vcf-files
+
+            // Gather vcfs and vcf-tbis for concatenating germline-vcfs
+            germline_vcfs_with_tbis = Channel.empty()
+
+            germline_vcfs_with_tbis = germline_vcfs_with_tbis.mix(
```

Comment:
would format this section a bit more densely

Reply:
Do you mind the linjes getting long?  Like 180 characters?